<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>histoDialog</class>
 <widget class="QDialog" name="histoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Equalization</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="21,0,19">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="15,84,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,4,12">
        <item>
         <widget class="QPushButton" name="ReadButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="channelBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="HistoWidget" name="histoView" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>3</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,24,0,11,0">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="title">
         <string>Channel Selection</string>
        </property>
        <widget class="QCheckBox" name="RCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Red</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="GCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Green</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="BCheck">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>191</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You must choose &lt;span style=&quot; font-weight:600;&quot;&gt;at least one&lt;/span&gt;&lt;br&gt; item for equalization.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OKButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="CancelButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoWidget</class>
   <extends>QWidget</extends>
   <header>histowidget.h</header>
   <slots>
    <slot>loadHistoData()</slot>
    <slot>displayHisto()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ReadButton</sender>
   <signal>clicked()</signal>
   <receiver>histoView</receiver>
   <slot>loadHistoData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channelBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>histoView</receiver>
   <slot>displayHisto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>histoDialog</receiver>
   <slot>checkMsg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>histoDialog</receiver>
   <slot>checkMsg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>histoDialog</receiver>
   <slot>checkMsg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>histoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>histoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>histoDialog</receiver>
   <slot>applyHisto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkMsg()</slot>
  <slot>applyHisto()</slot>
 </slots>
</ui>
